server/models/ireporterModel.js

```diff
@@ -0,0 +1,29 @@
+import { Pool } from 'pg';
+import env from 'dotenv';
+
+env.config();
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL,
+});
+
+
+export default {
+  /**
+   * DB Query
+   * @param {string} text
+   * @param {Array} params
+   * @returns {object} promise
+   */
+  query(text, params) {

```

Comment:
Please can you shed more light on this feedback, I don't really understand it

Reply:
I was actually using a lot of queries here. But since I started using this it actually helped me reduce my lines of code and helped me with a lot more manipulations. I actually saw this at codementor, and I realized it's usefulness